src/parquet/column/properties.h

```diff
@@ -162,14 +150,13 @@ class PARQUET_EXPORT WriterProperties {
     }
 
     std::shared_ptr<WriterProperties> build() {
-      return std::shared_ptr<WriterProperties>(new WriterProperties(allocator_,
-          dictionary_enabled_, dictionary_pagesize_, default_encoding_, encodings_,
-          pagesize_, version_, default_codec_, codecs_));
+      return std::shared_ptr<WriterProperties>(
+          new WriterProperties(allocator_, dictionary_pagesize_, default_encoding_,
+              encodings_, pagesize_, version_, default_codec_, codecs_));
     }
 
    private:
     MemoryAllocator* allocator_;
-    bool dictionary_enabled_;
```

Comment:
We currently only need the `encoding` parameter to determine how we encode the data. This was currently totally redundant information and I don't like to keep variables that are possibly needed (or not) for future implementation.